### DISCUSSION TITLE
[Ultima] Does Ultima support Nuxt 3

### DISCUSSION THREAD

**Discussion by codefl** [primefaces/community#1671] [PrimeVue Templates] [answered]
Hello,

I'm interested to purchase the Ultima template. I would like to check whether this template supports Nuxt 3.

Thanks,
Frank

**Comment by tugcekucukoglu** [comment, marked as answer]
Hello,

Ultima currently has Vite support. Since the Nuxt and Vite projects will be very similar, we can help you during the transition period. Apollo also has Nuxt support if you're interested.

Regards,

**Reply by codefl** [reply]
Thanks for the reply. I have chose to purchase the Apollo instead. Thanks
